examples/config-openid.json

```diff
@@ -0,0 +1,75 @@
+{
+  "distSpecVersion": "1.1.0-dev",
+  "storage": {
+    "rootDirectory": "/tmp/zot",
+    "dedupe": true
+  },
+  "http": {
+    "address": "127.0.0.1",
+    "port": "8080",
+    "realm": "zot",
+    "auth": {
+      "htpasswd": {
+        "path": "test/data/htpasswd"
+      },
+      "openid": {
```

Comment:
Mention what is mandatory in the docs website

Reply:
Opened https://github.com/project-zot/project-zot.github.io/issues/103 to track this. Please update that issue with any missing requirements.